_format/1.1/index.md

```diff
@@ -746,38 +746,19 @@ of this member **MUST** be an object (a "links object").
 <a href="#document-links-link" id="document-links-link"></a>
 Within this object, a link **MUST** be represented as either:
 
-* a URI-reference [[RFC3986 Section 4.1](https://tools.ietf.org/html/rfc3986#section-4.1)] to the link's target.
-* <a id="document-links-link-object"></a>an object ("link object") which **MAY**
-  contain the following members:
-  * `href`: a URI-reference [[RFC3986 Section 4.1](https://tools.ietf.org/html/rfc3986#section-4.1)] to the link's target.
-  * `rel`: a [link relation type][link relation type] defining the semantics of
-    the link.
-  * `anchor`: a URI-reference [[RFC3986 Section 4.1](https://tools.ietf.org/html/rfc3986#section-4.1)]
-    to the link's context. By default, the context of a link is the
-    [top-level object][top level], [resource object][resource objects], or
-    [relationship object][relationships] in which the link appears.
-  * `describedby`: a [link] to a description document (e.g. OpenAPI or JSON
-    Schema) for the link target.
-  * `title`: a string which serves as a label for the destination of a link
-    such that it can be used as a human-readable identifier (e.g., a menu
-    entry) in the language indicated by the `Content-Language` header field (if
-    present).
-  * `type`: a string a which serves as a hint indicating the media type of the
-    target resource.
-  * `hreflang`: a string or an array of strings which serve as a hint
-    indicating the language of target resource. An array of strings indicates
-    that multiple languages of the target resource are available. Each string
-    **MUST** be a valid language tag
-    [[RFC5646](https://tools.ietf.org/html/rfc5646)].
-  * `meta`: a meta object containing non-standard meta-information about the
-    link.
+* a string whose value is a URI-reference [[RFC3986 Section 4.1](https://tools.ietf.org/html/rfc3986#section-4.1)]
+  pointing to the link's target.
+* a [link object]
 
-> Note: the `type` and `hreflang` members are only hints; the target resource
-> is not guaranteed to be available in the indicated media type or language
-> when the link is actually followed.
+A link's relation type **SHOULD** be inferred from the name of the link unless a
+`rel` member is present on the [link object].
```

Comment:
perhaps:

> A link's relation type **SHOULD** be inferred from the name of the link unless it's represented by a [link object] with a `rel` member.

Reply:
Sorry, I pushed a commit while you were reviewing this. WDYT of the current wording?

Reply:
Yeah, I think that works well 👍